### ISSUE TITLE
Show source website in the My Addons list

### ISSUE THREAD

**Issue by filliph** [jliddev/WowUp#28]
**TL;DR:** Either add the website the addon updates from next to the zip name like "AckisRecipeList-8.3.0.2.zip · Curse", or add a new column.

Currently, when searching for an addon, it'll tell you what site it's installing from. This is great, because sometimes we'd prefer to install from Curse compared to WoWInterface, etc.

This information is not available in the My Addons list. This is a UX problem for new users, since an addon will have multiple entries if it exists on multiple websites, forcing us to click on the entry to open the website instead of just being able to see the information at a glance.
